cartridge/test-helpers/stateboard.lua

```diff
@@ -0,0 +1,92 @@
+--- Extended luatest.Server class to run the stateboard.
+--
+-- @classmod cartridge.test-helpers.stateboard
+
+local checks = require('checks')
+local fio = require('fio')
+
+-- Defaults.
+local Stateboard = {
```

Comment:
It should be `luatest.Server:inherit({})`. See [test-helpers/server.lua](https://github.com/tarantool/cartridge/blob/master/cartridge/test-helpers/server.lua) implementation.